### PR TITLE
gr: use new DPI setting for SVG output

### PR DESCRIPTION
**Pull request by jheinen**

### CODE REVIEW

File: src/backends/gr.jl [hunk 1]
```diff
@@ -544,6 +544,9 @@ function gr_display(plt::Plot, fmt="")
 
     _gr_thickness_scaling[1] = plt[:thickness_scaling]
     dpi_factor = plt[:dpi] / Plots.DPI
+    if fmt == "svg" then
```
Comment (mkborregaard) [inline src/backends/gr.jl]: there's a stray `then` here which breaks this code.

**Comment by mkborregaard** [comment]
Hi @jheinen I can see you already merged this, but there's an issue with the Atom integration, where the size of the plot no longer adapts to the plot pane in Atom. @pfitzseb is that best to fix from the Atom side do you think?

File: src/backends/gr.jl (hunk 1 above)
Comment (mkborregaard) [reply src/backends/gr.jl]: I've pushed a fix for this now.

**Comment by jheinen** [comment]
The output is ok both in Jupyter and Atom. The `width` and `height` attributes in the SVG header don't change, only the `viewBox` (and all internal coordinates - of course) have a higher resolution (x4)

**Comment by mkborregaard** [comment]
This is how `heatmap(5,5)` looks in my Atom on master
skaermbillede 2018-09-07 kl 15 51 04

**Comment by jheinen** [comment]
Not for me - are you really using the 0.34.0 run-time?

**Comment by mkborregaard** [comment]
probably not, though I did the ENV+build thing

**Comment by mkborregaard** [comment]
Let me try to checkout GR master - maybe we should up the `REQUIRE` again?

**Comment by mkborregaard** [comment]
It worked on the new master :heart: So OK, let's update the GR require version and push a Release once the GR metadata one gets merged.

**Comment by mkborregaard** [comment]
Or, no, there's still the concern that @daschw mentioned?

**Comment by daschw** [comment]
> The width and height attributes in the SVG header don't change, only the viewBox (and all internal coordinates - of course) have a higher resolution (x4)

I guess that is alright for SVGs then, so my concern should be no problem with the new GR run-time.

**Pull request merged**